python/paddle/fluid/tests/unittests/dygraph_to_static/test_fetch_feed.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+from paddle.fluid.dygraph.jit import dygraph_to_static_output
+
+import numpy as np
+import unittest
+
+import paddle.fluid as fluid
+
+SEED = 2020
+
+
+class Pool2D(fluid.dygraph.Layer):
+    def __init__(self):
+        super(Pool2D, self).__init__()
+        self.pool2d = fluid.dygraph.Pool2D(
+            pool_size=2, pool_type='avg', pool_stride=1, global_pooling=False)
+
+    @dygraph_to_static_output
+    def forward(self, x):
+        inputs = fluid.dygraph.to_variable(x)
+        pre = self.pool2d(inputs)
+        return pre
+
+
+class Linear(fluid.dygraph.Layer):
+    def __init__(self):
+        super(Linear, self).__init__()
+
+    @dygraph_to_static_output
+    def forward(self, x):
+        fc = fluid.dygraph.Linear(
```

Comment:
A better way is needed to deal with the layers with parameters in you next PR.

Reply:
Thanks